toolkit/offchain/src/init_governance/tests.rs

```diff
@@ -48,7 +48,7 @@ fn expected_transaction() -> serde_json::Value {
 				{
 					"address": "addr_test1vpmd59ajuvm34d723r8q2qzyz9ylq0x9pygqn7vun8qgpkgs7y5hw",
 					"amount": {
-						"coin": "9919227878",
+						"coin": "1137840",
```

Comment:
These tests seems overspecified for me. Every little change causes they need to be updated.
There is added value: I can see that in this particular example number of outputs increased.